nixos/modules/services/x11/hardware/synaptics.nix

```
@@ -44,19 +44,19 @@ in {
       };
 
       accelFactor = mkOption {
-        type = types.nullOr types.string;
+        type = types.nullOr types.str;
```

this is more of a joke, but...
```suggestion
        type = with types; nullOr (coercedTo float (toString) str);
```

---

No that's a good suggestion, but I think I'll leave this out of scope for this PR